.github/workflows/cd-sql-engine-tests.yaml

```diff
@@ -107,6 +107,55 @@ jobs:
           MF_SQL_ENGINE_URL: ${{ secrets.MF_BIGQUERY_URL }}
           MF_SQL_ENGINE_PASSWORD: ${{ secrets.MF_BIGQUERY_PWD }}
 
+  postgres-tests:
```

Comment:
These have no special environment dependencies, so they fire immediately on adding the label.

Do we want to move this to the blocking test suite that runs on every PR? There are no external dependencies that GitHub can't handle on its own, we don't need any secrets, and PostgreSQL is pretty fast, so it might be nice to have a "regular" engine test running to make sure we don't forget plan file updates and whatnot.

Reply:
Yeah, it would be fine to add this, but DuckDB is also a "regular" engine.